src/__tests__/__snapshots__/fjs_babel-test.js.snap

```diff
@@ -82,8 +81,7 @@ if (foo) {
   foobar();
 }
 
-if (foo) {
-}
+if (foo) {}
```

Comment:
Maybe we want to treat empty if blocks differently to normal blocks but for now it seems like a nicer default to collapse these.